### PR TITLE
Resolves #254 - allow custom JPCRM builds/versions

### LINKED ISSUE #254: Allow different versions of Jetpack CRM to be installed
In order to better test our code, we need to be able to install various versions of Jetpack CRM. This could include previous versions as well as custom builds.

### PR DESCRIPTION
**Pull request by tbradsha**
Resolves #254 - allow custom JPCRM builds/versions

In theory, this PR allows the following:

* install JPCRM from WP.org
* install a specific version of JPCRM from WP.org
* install a specific build of JPCRM (from our public S3 bucket)

~To my knowledge I'm not easily able to test the actual deploy/site creation.~ Edit: figured out how to test and looks like it works!

The `/specialops` page looks something like this, with the JPCRM settings toggling visibility:

image

### CODE REVIEW

File: features/jetpack-crm.php [hunk 1]
```diff
@@ -0,0 +1,153 @@
...
+		function ( $s ) use ( $cmd ) {
+			return "$s && $cmd";
+		}
+	);
+}
+
+/**
+ * Register a shortcode which renders Jetpack Licensing controls suitable for SpecialOps usage.
+ */
+\add_shortcode(
```
Comment (cleacos) [inline features/jetpack-crm.php:130]: And this backslash?

**Review by cleacos — COMMENTED** [review]
For me it is okay. I left a comment. Other than that... let's try it...

File: features/jetpack-crm.php (hunk 1 above)
Comment (tbradsha) [reply features/jetpack-crm.php:130]: Global namespace; just matching the style used elsewhere.

Comment (cleacos) [reply features/jetpack-crm.php:130]: Should we do the same for `add_action` in line #10?

**Comment by kraftbj** [comment]
From .org and from a previous version works. I didn't try from S3, but if y'all said it works, let's do it.

**Pull request merged**